build.py

```diff
@@ -1,19 +1,39 @@
+import os
+import shlex
+import shutil
 import subprocess
+import zipfile
 
 from pathlib import Path
 
 
-def meson(*args):
-    subprocess.call(["meson", *list(args)])
+def maturin(*args):
+    subprocess.call(["maturin", *list(args)])
 
 
 def _build():
     build_dir = Path(__file__).parent.joinpath("build")
     build_dir.mkdir(parents=True, exist_ok=True)
 
-    meson("setup", build_dir.as_posix())
-    meson("compile", "-C", build_dir.as_posix())
-    meson("install", "-C", build_dir.as_posix())
+    wheels_dir = Path(__file__).parent.joinpath("target/wheels")
```

Comment:
I wonder if we should add cleanup/deletion of maturin-built wheels.

Reply:
I added the cleanup at the end of the build step